.github/workflows/OCV-PR-4.x-W10-UWP.yaml

```diff
@@ -0,0 +1,71 @@
+name: OCV PR:4.x W10 UWP
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '.github/workflows/OCV-PR-4.x-W10-UWP.yaml'
+  workflow_call:
+
+concurrency:
+  group: OCV-PR-4.x-W10-UWP-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  EXTRA_CMAKE_OPTIONS: '-DOPENCV_DOWNLOAD_PATH=%BINARIES_CACHE% -DBUILD_opencv_gapi=OFF -DWITH_MSMF=OFF -DBUILD_TESTS=OFF -DBUILD_PERF_TESTS=OFF -DBUILD_EXAMPLES=ON -DCMAKE_BUILD_TYPE=Release'
```

Comment:
`-DWITH_MSMF=OFF` is not correct. It should be the only back-end for UWP, if I understand correctly.
Also we can disable samples for UWP build as console apps are not relevant for the configuration.

Reply:
There is separate WinRT backend, regular MSMF can not be built due to errors, thus I've disabed it. I'll disable samples and apps build.